source/agora/consensus/EnrollmentManager.d

```diff
@@ -287,12 +330,12 @@ public class EnrollmentManager
         this.enroll_key = frozen_utxo_hash;
 
         // N, cycle length
-        // Note, this name iis misleading, as a cycle here is the length
+        // Note, this name is misleading, as a cycle here is the length
```

Comment:
It looks like you introduced the spelling mistake in the previous commit, and fixed it in this commit.